packages/core/EventManager.ts

```diff
@@ -172,7 +172,10 @@ export default class EventManager {
     }
 
     // Update the calendar event with the proper video call data
-    results.push(...(await this.updateAllCalendarEvents(evt, booking)));
+    const calendarReference = booking.references.filter((reference) => reference.type.includes("_calendar"))[0];
```

Comment:
If we're using the first one we can just use `find` then. It will return the first item found.
```suggestion
    const calendarReference = booking.references.find((reference) => reference.type.includes("_calendar"));
```